mongo/topic_store.go

```diff
@@ -107,9 +107,6 @@ func (m *Mongo) CheckTopicExists(ctx context.Context, id string) error {
 		Find(bson.M{"id": id}).
 		Count(ctx)
 	if err != nil {
-		if dpMongoDriver.IsErrNoDocumentFound(err) {
```

Comment:
Please put these lines back in as the return code pf 'ErrTopicNotFound' is required.

Reply:
have put it back